example/zipkin/main.go

```diff
@@ -46,25 +48,32 @@ func initTracer(url string) {
 	if err != nil {
 		log.Fatal(err)
 	}
+
+	batcher := sdktrace.NewBatchSpanProcessor(exporter)
+
+	tp := sdktrace.NewTracerProvider(sdktrace.WithSpanProcessor(batcher))
+	global.SetTracerProvider(tp)
+
+	return func() {
+		tp.Shutdown(context.Background())
+	}
 }
 
 func main() {
 	url := flag.String("zipkin", "http://localhost:9411/api/v2/spans", "zipkin url")
 	flag.Parse()
 
-	initTracer(*url)
+	flusher := initTracer(*url)
+	defer flusher()
```

Comment:
```suggestion
	shutdown := initTracer(*url)
	defer shutdown()
```

Since this is actually shutting down the provider let's be explicit about that.